documentation/FAQ.md

```diff
@@ -1,6 +1,30 @@
 Frequently Asked Questions
 ==========================
 
+* On MacOS M1 if you see any of the following errors:
+    ```
+    (lldb) plugin load libsosplugin.so
+    error: this file does not represent a loadable dylib
+    ```
+    or if the sos plugin loads correct but you see this error on any command:
+    ```
+    (lldb) setsymbolserver -ms
+    Error: Fail to initialize coreclr 80070008
+    ```
+    or something like this error:
+    ```
+    (lldb) setsymbolserver -ms
+    SOS_HOSTING: Fail to initialize hosting runtime '/usr/local/share/dotnet/shared/Microsoft.NETCore.App/6.0.29/libcoreclr.dylib' 80004005
+    Unrecognized command 'setsymbolserver' because managed hosting failed or was disabled. See sethostruntime command for details.
+    ```
+
+    Try the following workaround for Apple's standard lldb installation:
+    ```
+    $ sudo cp /Applications/Xcode.app/Contents/Developer/usr/bin/lldb /usr/local/bin
+    $ sudo install_name_tool -add_rpath /Applications/Xcode.app/Contents/SharedFrameworks /usr/local/bin/lldb
+    $ sudo codesign --force --sign - /usr/local/bin/lldb
+    ```
+
```

Comment:
```suggestion

This will make a copy of Xcode lldb and adhoc sign it. Note that this workaround is only applicable to Xcode lldb, not the System lldb (`/usr/bin/lldb`), which may still fail to load. You may need to delete or rename `~/.lldbinit` to use `/usr/bin/lldb`.

```

System lldb is different stripped version, and it is available even without the Xcode. Something which Apple folks maybe able to sort out, unless they want to intentionally block the third-party plugins.